src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -378,6 +378,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Function
 
         <Extension>
+        Friend Function GetDeclaringSyntaxNode(Of T As VisualBasicSyntaxNode)(this As SourceMemberContainerTypeSymbol) As T
+
+            For Each node In this.DeclaringSyntaxReferences.Select(Function(d) d.GetSyntax())
```

Comment:
It only differs in the assertion. I merged them back with a special assertion case for `SourceMemberContainerTypeSymbol`.

---
In reply to: [93084697](https://github.com/dotnet/roslyn/pull/15809#discussion_r93084697) [](ancestors = 93084697)